ui/components/MesheryVisualizeComponent.js

```diff
@@ -11,9 +15,60 @@ class MesheryVisualizeComponent extends React.Component {
       { data: { source: 'one', target: 'two', label: 'Edge from Node1 to Node2' } }
     ];
 
+    let cxtMenuSettings = {
+      menuRadius: 100, // the radius of the circular menu in pixels
+      selector: 'node', // elements matching this Cytoscape.js selector will trigger cxtmenus
+      commands: [ // an array of commands to list in the menu or a function that returns the array
+        {
+          fillColor: 'rgba(200, 200, 200, 0.75)', // optional: custom background color for item
```

Comment:
Depending upon how we bring Meshery and MeshMap together, this can reference a global variable.